ceph-medic-pull-requests/config/definitions/ceph-medic-pull-requests.yml

```diff
@@ -0,0 +1,64 @@
+- scm:
+    name: ceph-medic 
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-medic
+          branches:
+            - ${sha1}
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          basedir: "ceph-medic"
+          skip-tag: true
+          wipe-workspace: true
+
+
+- job:
+    name: ceph-medic-pull-requests
+    description: Runs tox tests for ceph-medic on each GitHub PR
+    project-type: freestyle
+    node: trusty && small
```

Comment:
There is a ``py27`` tox environment as well, why would we not test that as well?